src/earn/poolInfo.ts

```diff
@@ -32,3 +33,32 @@ export async function fetchAavePoolInfo(assetAddress: Address) {
     throw err
   }
 }
+
+export async function fetchAavePoolUserBalance({
+  tokenAddress,
+  walletAddress,
+}: {
+  tokenAddress: Address
+  walletAddress: Address
+}) {
+  try {
+    Logger.debug(TAG, 'Fetching Aave pool user balance', { tokenAddress, walletAddress })
+    const aaveUSDCContract = getContract({
+      abi: erc20.abi,
+      address: tokenAddress,
+      client: {
+        public: publicClient[Network.Arbitrum],
+      },
+    })
+
+    const balance = await aaveUSDCContract.read.balanceOf([walletAddress])
+    const decimals = await aaveUSDCContract.read.decimals()
```

Comment:
the arb usdc token is part of address metadata right? If we have the tokenId can't we just get this by doing `useTokenId` hook instead of making another contract call here?